tiatoolbox/dataloader/wsireader.py

```diff
@@ -101,7 +235,7 @@ def read_region(self, start_w, start_h, end_w, end_h, level=0):
         """
         raise NotImplementedError
 
-    def slide_thumbnail(self):
+    def slide_thumbnail(self, scale=1.25, units="power"):
```

Comment:
What is the difference if `units` is sthg else ?

Reply:
as in `mpp` `baseline` etc. as in previous function

Reply:
For example `scale=30` and `units="mpp"` would produce a thumbnail at 30 microns per pixel. 

Scale has now been renamed to resolution. See [`read_rect`](https://tia-toolbox.readthedocs.io/en/feature-calc-scale/usage.html#tiatoolbox.dataloader.wsireader.OpenSlideWSIReader.read_rect) for more (thumbnail uses read_rect).